HOWTO/DOCUMENTATION.md

```diff
@@ -443,7 +443,7 @@ Get information about a thing.
 As the actual version is generally not yet certain at the time when a
 change gets merged, the an internal ticket number (for example
 `OTP-12345`) to use as a placeholder in the respective `since` tags,
-such as `-doc #{since => ~"OTP OTP-12345"}.`.  When your new
+such as `-doc #{since => ~"OTP @OTP-12345@"}.`.  When your new
```

Comment:
@jchristgit It would be nice if you could fix the grammar of that sentence in this PR as well.